tests/__mocks__/parser/resolve.mock.js

```diff
@@ -0,0 +1,32 @@
+const parser = jest.genMockFromModule('../../../lib/parser/index.js');
+const path = require('path');
+
+function mockPromise(value) {
+	return (value || {}).then ? value : {
+		then: function (callback) {
+			return mockPromise(callback(value));
+		}
+	};
+}
+function spawnChild(pkg) {}
+
+function getLoc(option) {
```

Comment:
In that case, we should call it `getLocation`, a good practice when naming variables is not to save bytes in words, the goal is to have names that are right away understood and that are meaningful. A good guide on these series of good practices to write clean code, was recently translated into JS from a really good book from Robert Martin's book [*Clean Code*](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)

Here the link to it: https://github.com/ryanmcdermott/clean-code-javascript#variables

I understand that we want to do things quickly to setup the groundwork, however we should try to keep things as clean as possible, to avoid that the code grows unclean and later on is harder to clean.